contracts/mirror_mint/src/state.rs

```diff
@@ -50,6 +50,7 @@ pub struct AssetConfig {
     pub auction_discount: Decimal,
     pub min_collateral_ratio: Decimal,
     pub end_price: Option<Decimal>,
+    pub mint_end: Option<u64>,
```

Comment:
When we change the already existing data structure, we must update existing values to new data struct.

Ex) 
Read all existing AssetConfig and update that values to new AssetConfig with `None` `mint_end`
https://github.com/Anchor-Protocol/money-market-contracts/blob/bugfix/update-market-logics/contracts/market/src/migration.rs